### PR TITLE
Update padstack.py

### PR DESCRIPTION
**Pull request by PipKat**

### CODE REVIEW

File: pyaedt/edb_core/padstack.py [hunk 1]
```diff
@@ -64,7 +63,7 @@ def _layers(self):
         return self.parent.core_stackup.stackup_layers
     @property
     def padstacks(self):
-        """List all padstacks via padstack definitions.
+        """Retrive all padstacks.
```
Comment (PipKat) [inline pyaedt/edb_core/padstack.py:66]:
```suggestion
        """Retrieve all padstacks.
```

File: pyaedt/edb_core/padstack.py [hunk 2]
```diff
@@ -223,6 +219,30 @@ def create_padstack(self, padstackname=None, holediam='300um', paddiam='400um',
...
+        Parameters
+        ----------
+        position :
+        definition_name :
+        net_name : str, optional
+            Name of the net. The default is ``""``.
+        via_name : str, optional
+            The default is ``""``.
+        rotation : 
+            Rotation of the padstack in degress. The default
```
Comment (PipKat) [inline pyaedt/edb_core/padstack.py:233]:
```suggestion
            Rotation of the padstack in degrees. The default  
```

File: pyaedt/edb_core/padstack.py [hunk 3]
```diff
@@ -1,8 +1,7 @@
 """
-This module contains all EDB functionalities in the ``EDB Padstack`` class.
-
-
+This module contains the ``EdbPadstack`` class.
```
Comment (PipKat) [inline pyaedt/edb_core/padstack.py:2]:
```suggestion
This module contains the ``EdbPadstacks`` class.
```

File: pyaedt/edb_core/padstack.py [hunk 4]
```diff
@@ -64,7 +63,7 @@ def _layers(self):
         return self.parent.core_stackup.stackup_layers
     @property
```
Comment (akaszynski) [inline pyaedt/edb_core/padstack.py:64]:
```suggestion
        return self.parent.core_stackup.stackup_layers

    @property
```

File: pyaedt/edb_core/padstack.py [hunk 5]
```diff
@@ -223,6 +219,30 @@ def create_padstack(self, padstackname=None, holediam='300um', paddiam='400um',
     @aedt_exception_handler
     def place_padstack(self, position, definition_name, net_name='',
                        via_name="", rotation = 0, fromlayer = None, tolayer = None, solderlayer=None):
```
Comment (akaszynski) [inline pyaedt/edb_core/padstack.py:221]:
```suggestion
                       via_name="", rotation=0, fromlayer=None, tolayer=None, solderlayer=None):
```

File: pyaedt/edb_core/padstack.py [hunk 6]
```diff
@@ -223,6 +219,30 @@ def create_padstack(self, padstackname=None, holediam='300um', paddiam='400um',
...
+        
+        Parameters
+        ----------
+        position :
+        definition_name :
+        net_name : str, optional
+            Name of the net. The default is ``""``.
+        via_name : str, optional
+            The default is ``""``.
+        rotation :
```
Comment (akaszynski) [inline pyaedt/edb_core/padstack.py:232]:
```suggestion
        rotation : float, optional
```

File: pyaedt/edb_core/padstack.py [hunk 7]
```diff
@@ -223,6 +219,30 @@ def create_padstack(self, padstackname=None, holediam='300um', paddiam='400um',
     @aedt_exception_handler
     def place_padstack(self, position, definition_name, net_name='',
                        via_name="", rotation = 0, fromlayer = None, tolayer = None, solderlayer=None):
+        """Place the padstack.
+        
+        Parameters
+        ----------
+        position :
```
Comment (akaszynski) [inline pyaedt/edb_core/padstack.py:226]: @maxcapodi78, could you please help fill in the descriptions of these parameters?

**Review by akaszynski — CHANGES_REQUESTED** [review]
Need massimo to fill out the parameter descriptions.

File: pyaedt/edb_core/padstack.py (hunk 7 above)
Comment (maxcapodi78) [inline pyaedt/edb_core/padstack.py:226]:
```suggestion
        position : list
            list of [x,y] position where the via will be placed. x, y are float
```

**Review by maxcapodi78 — CHANGES_REQUESTED** [review]

File: pyaedt/edb_core/padstack.py [hunk 8]
```diff
@@ -62,9 +61,10 @@ def messenger(self):
     def _layers(self):
         """ """
         return self.parent.core_stackup.stackup_layers
+
     @property
     def padstacks(self):
-        """List all padstacks via padstack definitions.
+        """Retrieve all padstacks.
```
Comment (PipKat) [inline pyaedt/edb_core/padstack.py:67]:
```suggestion
        """List of padstacks via padstack definitions.
```

**Review by akaszynski — APPROVED** [review]
Thanks for fixing.  Hope you're fine with me merging this.

**Review by akaszynski — APPROVED** [review]

**Pull request merged**